src/components/nodes/messageFlow/messageFlow.vue

```diff
@@ -58,11 +65,13 @@ export default {
       return this.targetType === poolId;
     },
     targetIsInDifferentPool() {
+      const targetNodeType = this.targetNode.definition.$type;
```

Comment:
I like this extraction but do you think the definition of the variable should be closer to where it's used? How about having this on line 73?

Reply:
That's a good point.